### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by wuyepabdul**
In this milestone, I did the following:
- setup Bookstore app
- setup redux duck pattern
- create default Book component and Category component

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,21 +1,27 @@
-/* eslint-disable react/prop-types */
 import React from 'react';
-// import PropTypes from 'prop-types';
+import PropTypes from 'prop-types';
 
 const Book = ({ bookList }) => (
```
Comment (helman101) [inline src/components/Book.js]: - [x] This component should only display a single book data, and your `Books` component should display a list of `Book` components, kindly make this change 😄

File: src/components/Form.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      </div>
+      <div className="form-input select-input">
+        <select className="input-field">
+          <option>category</option>
+          <option>category</option>
+          <option>category</option>
+          <option>category</option>
+        </select>
+      </div>
+    </form>
```
Comment (helman101) [inline src/components/Form.js]: - [x] Your form should have 2 inputs, one for book `title` and one for book `author`, the category input is not necessary 🚀

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @wuyepabdul 👋, 

Nice job so far with this project 😄, but there are still some changes to do.

## Required Changes ♻️

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @wuyepabdul, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**